README.md

```diff
@@ -71,6 +71,23 @@ docker run --rm -e SCAN_URL="https://www.google.com/" lirantal/is-website-vulner
 
 :warning: A modern version of Chrome is assumed to be available when using `is-website-vulnerable`. It may not be safe to assume that this is satisfied automatically on some CI services. For example, [additional configuration](https://docs.travis-ci.com/user/chrome#selecting-a-chrome-version) is necessary for [Travis CI](https://travis-ci.com/).
 
+# Github Action
+Create .github/workflows/is-website-vulnerable.yml with the url that you want scanned:
+
+```
+name: Test site for publicly known js vulnerabilities
+
+on: push
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test for public javascript library vulnerabilities 
+        uses: lirantal/is-website-vulnerable@v1
```

Comment:
@mattorb what does version 1 point to here?
should we instead use `latest` ?

Reply:
Yes, for the purpose of the example, let's tie this to latest, rather than a tag, branch, or  release.   We can do that here with '@master'.  I will adjust it.

Reply:
(it is referencing the github repo, not the docker hub location)

Reply:
@mattorb the `lirantal/is-website-vulnerable` references the GitHub, right? so far I get that. The version though of `v1` also references the GitHub releases, right? we're now at 1.x something released so it's a bit ambiguous as to whether it will semver and get the latest 1.x or is it getting just the 1.0.0 version if it exists?

I think it is ok to change to just `@master` for now

Reply:
✔️  updated to '@master'.

Reply:
👍